libs/moment-adapter/karma.conf.js

```diff
@@ -0,0 +1,17 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+
+const { join } = require('path');
+const getBaseKarmaConfig = require('../../karma.conf');
+
+module.exports = function (config) {
+    const baseConfig = getBaseKarmaConfig();
+    config.set({
+        ...baseConfig,
+        coverageIstanbulReporter: {
+            ...baseConfig.coverageIstanbulReporter,
+            dir: join(__dirname, '../../coverage/libs/core')
```

Comment:
path should be '../../coverage/libs/moment-adapter'

Reply:
thanks